### ISSUE TITLE
Getting Error in Console response “Uncaught (in promise) null”

### ISSUE THREAD

**Issue by nisargrthakkar** [szchenghuang/react-google-invisible-recaptcha#30]
In the console, I am getting the following error

![captcha](https://user-images.githubusercontent.com/26029722/54812417-1662b200-4cb1-11e9-824b-91d351764894.png)

**Comment by BramDecuypere** [comment]
Having the same issue! Were you able to solve it? @nisargrthakkar

**Comment by nisargrthakkar** [comment]
@BramDecuypere  No, We are facing the same issue. In Google ReCaptcha issue is open https://github.com/google/recaptcha/issues/269. 

Personally, I don't think this issue is not related to low speed.

**Comment by blindfish3** [comment]
@nisargrthakkar - throttling and network conditions are absolutely at play here.  There's an unhandled timeout in Google's recaptcha code; and no easy way to handle it from outside.  If a timeout occurs - most often because of temporary network failure on the client end - the uncaught promise is logged.

I know this because I was tasked with dealing with the many Sentry errors this was generating on a React application.  IIRC I had to suppress them in Sentry :/

**Comment by trickydisco78** [comment]
Looks like someone has found a fix for this

https://github.com/google/recaptcha/issues/269#issuecomment-606838861
https://gist.github.com/ANTOSzbk/75ed7003e914162550f61399122a3bd4

Could this be implemented?

**Comment by npearson72** [comment]
I've noticed that is not happening in production / without React strict mode. Can anyone else confirm?

**Comment by Zues-12** [comment]
> I've noticed that is not happening in production / without React strict mode. Can anyone else confirm?

Nope, still happens to me

**Comment by ltmervin123** [comment]
I face the same issue when the ReCAPTCHA from react-google-recaptcha is rendered for a long period of time, even if it generates a valid token.

How do you guys resolve this issue?
